cognite/neat/rules/models/_rules/_types/_field.py

```diff
@@ -259,9 +261,9 @@ def _semantic_value_type_serializer(value: Any) -> str:
 CdfValueType = Annotated[
     DMSValueType | ViewPropEntity,
     BeforeValidator(
-        lambda value: DMS_VALUE_TYPE_MAPPINGS[value]
-        if value in DMS_VALUE_TYPE_MAPPINGS
-        else ViewPropEntity.from_raw(value)
+        lambda value: (
+            DMS_VALUE_TYPE_MAPPINGS[value] if value in DMS_VALUE_TYPE_MAPPINGS else ViewPropEntity.from_raw(value)
```

Comment:
Shouldn't there be a lower here?
```suggestion
            DMS_VALUE_TYPE_MAPPINGS[value] if value.lower() in DMS_VALUE_TYPE_MAPPINGS else ViewPropEntity.from_raw(value)
```